keyboards/ergosaurus/readme.md

```diff
@@ -0,0 +1,15 @@
+# ergosaurus
+
+![ergosaurus](imgur.com image replace me!)
```

Comment:
There should be an image link here.

Reply:
So strange, I know I added this - must've forgot to hit save! I'll get that pushed up today, thanks for catching!